jobserver/management/commands/create_user.py

```diff
@@ -21,12 +21,14 @@ def add_arguments(self, parser):
         parser.add_argument("--email", help="Defaults to username@example.com")
         parser.add_argument("--name", help="Defaults to username")
         parser.add_argument(
+            "-o",
             "--output-checker",
             action="store_true",
             help="Make user global OutputChecker",
         )
         parser.add_argument(
-            "--core-developer",
+            "-s",
+            "--staff-area-administrator",
```

Comment:
Looking back at the history of this management command...

```sh
git log --oneline -- jobserver/management/commands/create_user.py
```

...I see that Simon created it in 989c000f. It's not clear whether Simon intended to use it in scripts, but if he did, then this commit will break them. Could you drop into #team-rap and let them know?

Reply:
Yes, we use to script set up for a local job-server for integration testing.

https://github.com/opensafely-core/airlock/blob/main/job-server/local-setup.sh#L64

However, this scripting is covered in CI, so this change should detect that and our new failing workflows dashboard should show it, so lets see if it all works as intended!

Reply:
I guess I saw this more as a breaking API change, and then wondered whether there were any non-human clients.